### ISSUE TITLE
Create a WebSocket class to get game data in real time

### ISSUE THREAD

**Issue by alvarofpp** [bonsancon/pyblaze#2]
- URL: `wss://api-v2.blaze.com/replication/?EIO=3&transport=websocket`

**Comment by owati** [comment]
I will like to take up this issue. Just to clarify, a websocket class to get the real time date and a probably a method on the Baseclient class to call it..

**Comment by alvarofpp** [comment]
@owati Yeah. You can add an attribute in `BaseClient` to use the WebSocker class. But it is necessary to differentiate between the Double events and Crash events.

**Comment by owati** [comment]
Okay, that is clear...I will like to take it up..
